.buildkite/run.sh

```diff
@@ -33,7 +33,11 @@ cp .buildkite/.gitconfig "$HOME/"
 cat "$HOME/.gitconfig"
 
 echo "--- Run proxy docs"
-(cd proxy && time cargo doc --no-deps)
+(
+  cd proxy
+  export RUSTDOCFLAGS="-D intra-doc-link-resolution-failure"
+  time cargo doc --workspace --no-deps --all-features --document-private-items
```

Comment:
We don't wanna do `&&` here?

Reply:
`&&` is not needed. Just like the main shell for this script the subshell (instantiated by `( ... )`) has the `set -e` option so that any failing command will abort the script.